README.md

```diff
@@ -19,6 +19,10 @@ npm i react-native-mock-render --save-dev
 require('react-native-mock-render/mock'); // <-- side-effects!!!
 ```
 
+### Jest
+
+Jest [don't support](https://github.com/Root-App/react-native-mock-render/issues/23) `require.cache`, so the correct way to do the mocking is using `jest.mock('react-native', () => require('react-native-mock-render'), {virtual: true})` at your tests setup file.
```

Comment:
Can you change "don't support" to "doesn't support" and "at your tests setup file" to "in your test setup file"?

Reply:
Done, only I have left `tests` in plural because, well, it's a single setup file for several tests :-D The other spellchecks are a sympthom I'm tired :-P